packages/nitro-protocol/contracts/MultiAssetHolder.sol

```diff
@@ -541,6 +541,29 @@ contract MultiAssetHolder is IMultiAssetHolder, ForceMove {
         );
     }
 
+    /**
+     * @notice Checks that a given channel is in the Finalized mode.
+     * @dev Checks that a given channel is in the Challenge mode.
+     * @param channelId Unique identifier for a channel.
+     */
+    function _requireChannelFinalized(bytes32 channelId) internal view {
+        require(_mode(channelId) == ChannelMode.Finalized, 'Channel not finalized.');
+    }
+
+    function _updateFingerprint(
+        bytes32 channelId,
+        bytes32 stateHash,
+        address challengerAddress,
+        bytes32 outcomeHash
+    ) internal {
+        (uint48 turnNumRecord, uint48 finalizesAt, ) = _unpackStatus(channelId);
+
+        bytes32 newStatus = _generateStatus(
+            ChannelData(turnNumRecord, finalizesAt, stateHash, challengerAddress, outcomeHash)
+        );
+        statusOf[channelId] = newStatus;
+    }
```

Comment:
does this one belong in the `StatusManager`? Given that `_generateFingerprint` and friends are there?

Reply:
This is a subjective call, but I decided to place things in order to minimize the size of each contract's API (including internal / private API). Putting this in StatusManager includes it in ForceMove unnecessarily, which is 'bad'. Leaving it where it is now separates it from similar functions which is also 'bad'.

In my mind, reducing API size is more "objectively good" than grouping the similar functions.